testing/cockroach/force_autoreconf.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile
++++ b/Makefile
+@@ -217,6 +217,8 @@
+ # of them (or their dependencies) change.
+ .bootstrap: $(GITHOOKS) $(GLOCK) GLOCKFILE
+ 	@unset GIT_WORK_TREE; $(GLOCK) sync github.com/cockroachdb/cockroach
++	#Force autoconf to run, otherwise won't work on musl for instance
```

Comment:
Could you investigate this further and detail to us (cockroach labs) what the issue is. We'd be keen to ensure the build goes smoothly with musl too.